.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug in this library
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+This form is for bug reports for this library only.
+For general, language-agnostic questions and troubleshooting, please ask/look for answers [here](https://groups.google.com/g/adwords-api).
```

Comment:
I would add "questions and troubleshooting about the **Google Ads API**". I think this is important because the group name in the provided URL is "AdWords API" which will make it confusing for people.

Reply:
@devchas Could you confirm?